03b_model/src/run_model.py

```diff
@@ -157,50 +282,160 @@ def prep_input_target_data(inputs_xarray, target_xarray,
     with open(os.path.join(out_dir,'prepped_model_io_data'), 'wb') as handle:
         pickle.dump(prepped_model_io_data, handle)
 
-def write_model_params(out_dir, run_id, inputs, n_epochs_pre,
-                       learn_rate_pre, seq_len, hidden_units,
+def write_model_params(out_dir, run_id, inputs, n_epochs,
+                       learn_rate, seq_len, hidden_units,
+                       recur_dropout, dropout,
                        train_start_date, train_end_date,
                        val_start_date, val_end_date,
-                       test_start_date, test_end_date):
+                       test_start_date, test_end_date, inc_ante):
+    '''
+    Write model parameters to directory where model results will be saved
+
+    Parameters
+    ----------
+    out_dir : str
+        directory where model results will be saved
+    run_id : str
+        sub-directory within out_dir where model results will be saved
```

Comment:
What happens if you use the same run_id for 2 runs? Will the 1st be overwritten, or will the code stop when it detects data in these directories?

Reply:
It will be overwritten, this is something that I need to add back in. You can see that I added some code to do this lines 344-351, but it was producing odd behavior with snakemake so I commented it out. I will add it back in and try to troubleshoot the behavior

Reply:
I tried this out and it errored on creation of the snakemake file. Switching to a different run_id works.
```
WorkflowError:
Error recording metadata for finished job ([WinError 183] Cannot create a file when that file already exists: '\\.snakemake\\metadata\\tmpwwcy1f56.MDNiX21v' -> '\\.snakemake\\metadata\\MDNiX21vZGVsL291dC9pbnB1dHMuemFycg=='). Please ensure write permissions for the directory C:\Users\jsmith\OneDrive - DOI\Documents\Projects\DRB-Esal\CodeReviews\Galen\drb-estuary-salinity-ml\.snakemake
```